client/src/PlayerLanding/PlayerInGame.tsx

```diff
@@ -155,6 +157,10 @@ export function PlayerInGame() {
               </Text>
             </Flex>
 
+            <Flex justify="center">
```

Comment:
Do we really need this?  Can people really not divide by 2????

Reply:
UI 101 (trust me I'm an expert): Never expect your users to math

Reply:
See, in UI101 they teach you only oversimfilications.  Once you get into your grad studies, you realize that the key to UI design is fukkem